core/cron.py

```diff
@@ -0,0 +1,24 @@
+""" This module has all core app corn jobs"""
+
+from datetime import datetime
+
+from templated_mail.mail import BaseEmailMessage
+
+from core.models import BookLoan
+
+
+def email_overdue_books():
+    """cron job for emailing user which have a book overdue"""
+    today = datetime.now()
+    loan_queryset = BookLoan.objects.select_related("book").select_related('user').filter(date_due__lt=today)
```

Comment:
```suggestion
    loan_queryset = BookLoan.objects.select_related('book', 'user').filter(date_due__lt=today)
```

either use single quote `'` or double quote `"`